pennylane/measurements/measurements.py

```diff
@@ -128,13 +129,15 @@ class MeasurementProcess(ABC, metaclass=qml.capture.ABCCaptureMeta):
 
     # pylint:disable=too-many-instance-attributes
 
+    _shortname = None
```

Comment:
Just to make sure I understand, is `_shortname` going to `Optional[str]` or `Optional[str | ObservableReturnType]`. Because we're wanting to get rid of `ObservableReturnType`.

Reply:
Sorry about the miscommunication, but I was thinking this could be a strictly private string rep we can use in the repr.